src/css/components/panel.scss

```diff
@@ -233,6 +227,18 @@ $panel-padding: $grid-3;
 
 // a couple random panel styles
 
+.route-form-actions {
+
+  .panel-actions {
+    display: block;
+    text-align: left;
+
+    button:first-child {
+      margin-left: 0;
+      }
+    }
+}
+
```

Comment:
It was doing this:
![screen shot 2017-01-06 at 2 45 41 pm](https://cloud.githubusercontent.com/assets/1449852/21730697/dacd5720-d41e-11e6-94c8-2300491a80ce.png)

Now the buttons are aligned right and don't have as much space between the buttons.